index.js

```diff
@@ -379,17 +387,19 @@ class ClientServiceBase {
 
   /**
    * @param {string} assetId
-   * @param {number} age
+   * @param {number} startAge
+   * @param {number} endAge
    * @param {string} contractId
    */
-  async _validateLedgerWithContractExecution(assetId, age, contractId) {
+  async _validateLedgerWithContractExecution(
+      assetId, startAge, endAge, contractId,
+  ) {
     const argument = {
       'asset_id': assetId,
     };
 
-    if (age !== 0x7fffffff) { // Integer.MAX_VALUE in Java
-      argument['age'] = age;
-    }
```

Comment:
Don't need this part anymore. The argument validation is done in the upper layer in function `validateLedger`.